Content.Shared.Database/LogType.cs

```diff
@@ -81,4 +81,6 @@ public enum LogType
     Stamina = 76,
     EntitySpawn = 77,
     Container = 78,
+    Teleport = 79,
+    EntityDelete = 80,
```

Comment:
EntityDelete is too generic, I'd just make it Portal.

Reply:
entitydelete is fine i think but its weird if literally nothing else uses it